src/Core/AbstractApi.php

```diff
@@ -91,16 +95,30 @@ public function __construct($configuration = [], ?CredentialProvider $credential
     abstract protected function getSignatureScopeName(): string;
 
     /**
-     * @param string[]|string[][]      $headers headers names provided as keys or as part of values
-     * @param string|resource|callable $body
+     * @param string[]|string[][]                    $headers headers names provided as keys or as part of values
+     * @param string|resource|callable|iterable|null $body
      */
     final protected function getResponse(string $method, $body, $headers = [], ?string $endpoint = null): ResponseInterface
```

Comment:
I wonder if it shouldn't be `Stream $body`. This thing will be more efficient if performed by the called

Reply:
That require that we update all `requestBody` functions.

Reply:
Hm.. Currently, we can mark all Streams as `@internal`. Is that a good thing?

Reply:
Let's move `Stream` to `@internal`, and we can remove it later if we want.
I also set classes final